sympy/solvers/solvers.py

```diff
@@ -459,11 +474,22 @@ def solve(f, *symbols, **flags):
         >>> solve([x - 3, y - 1], dict=True)
         [{x: 3, y: 1}]
 
+     
+    ** to get only desired value value if dict=True
```

Comment:
There's no need to duplicate the example here. Just add an example to the above of accessing the solution. Something like

```
>>> sol = solve([x - 3, y - 1], dict=True)
>>> sol
[{x: 3, y: 1}]
>>> sol[0][x]
3
```